specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -4588,11 +4588,21 @@
         "source": {
           "type": "string",
           "description": "The source of the input."
+        },
+        "sourceContext": {
+          "type": "string",
+          "description": "The source context used for selecting recursive inputs."
+        },
+        "inputs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InputFieldMappingEntry"
+          },
+          "description": "The recursive inputs used when creating a complex type."
         }
       },
       "required": [
-        "name",
-        "source"
```

Comment:
Removing required field in a stable version usually is breaking change.

Reply:
Yes, this is followed by a SDK change where we have revved up the major version.
i.e., We know it's a breaking change and are ok taking it

Reply:
There are two types of breaking changes, SDK and Rest API. We don't allow to make Rest API breaking changes in the same API version.